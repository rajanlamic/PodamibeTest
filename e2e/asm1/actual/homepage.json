{
  "page": {
    "pageInfo": {
      "pageName": "homePage"
    },
    "events": []
  },
  "event": [],
  "user": {
    "profile": {
      "profileInfo": {
        "websphereId": ""
      }
    }
  }
}