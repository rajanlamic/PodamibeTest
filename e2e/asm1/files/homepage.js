module.exports  = {
  "KF$page": {
    "KF$pageInfo": {
      "KF$VF$pageName": "homePage"
    },
    "KF$events": []
  },
  //"KA$optKey" : [],
  "KF$event": [],
  "KF$user": {
    "KF$profile": {
      "KF$profileInfo": {
        "KF$VA$websphereId": "1444230441570-6"
      }
    }
  }
}