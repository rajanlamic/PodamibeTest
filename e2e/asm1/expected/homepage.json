{
  "KF.page": {
    "KF.pageInfo": {
      "KF.pageName": "VF.homePage"
    },
    "KF.VF.events": []
  },
  "KF.VF.event": [],
  "KF.user": {
    "KF.profile": {
      "KF.profileInfo": {
        "KF.websphereId": "VA.1444230441570-6"
      }
    }
  }
}